play/src/containers/GameSessionContainer.tsx

```diff
@@ -16,15 +16,81 @@ export default function GameSessionContainer() {
     ) as IGameSession
   );
   const [teamAvatar, setTeamAvatar] = useState(0); // eslint-disable-line @typescript-eslint/no-unused-vars
-  const [joinGameState, setjoinGameState] = useState<JoinGameState>(JoinGameState.SELECTAVATAR);
+  const [joinGameState, setjoinGameState] = useState<JoinGameState>(JoinGameState.SPLASHSCREEN);
 
 
+  const handleOptionChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = Number(event.target.value);
+    switch (value) {
+      case 0:
+        setjoinGameState(JoinGameState.SPLASHSCREEN);
+        break;
+      case 1:
+        setjoinGameState(JoinGameState.ENTERGAMECODE);
+        break;
+      case 2:
+        setjoinGameState(JoinGameState.ENTERNAME);
+        break;
+      case 3:
+        setjoinGameState(JoinGameState.SELECTAVATAR);
+        break;
+      case 4:
+        setjoinGameState(JoinGameState.HOWTOPLAY);
+        break;
+      default:
+        // Handle invalid value here
+        break;
+    }
```

Comment:
In Typescript, you can assign values to enums, any reason we don't use that approach instead of having enum and then a separate value set for each?

Reply:
Fair point. this was just the debug radio controls for review though so I've removed all the code from the PR.